wled00/ir_codes.h

```diff
@@ -93,3 +186,13 @@
 #define COLOR_MAGENTA   0xFF00DC
 #define COLOR_PINK      0xFF00A0
 #define COLOR_WHITE     0xFFFFDC
+#define COLOR_WARMWHITE2     0xFFAA69
+#define COLOR_WARMWHITE      0xFFBF8E 
+#define COLOR_NEUTRALWHITE   0xFFD4B4
+#define COLOR_COLDWHITE      0xFFE9D9
+#define COLOR_COLDWHITE2     0xFFFFFF
+#define COLOR2_WARMWHITE2    0xFFFF9900
+#define COLOR2_WARMWHITE     0xFF825A00 
+#define COLOR2_NEUTRALWHITE  0xFF000000
+#define COLOR2_COLDWHITE     0xFF7F7F7F
+#define COLOR2_COLDWHITE2    0xFFFFFFFF
```

Comment:
more colors - the ones that begin with "COLOR2_" are 32 bit / RGBW - the other ones 24 bit / RGB.